frontend/src/app/about/Creators.tsx

```diff
@@ -0,0 +1,57 @@
+import styles from './page.module.css'
+import Image from 'next/image'
+import charlottePic from './profile-photos/charlotte.png'
+import nadiaPic from './profile-photos/nadia.jpeg'
+import jessPic from './profile-photos/jess.jpg'
+
+export default function Creators () {
+    return (
+        <section className={styles.about_the_creators_container}>
+          <h2
+            className={`${styles.subheading} ${styles.about_the_creators_header}`}>
+            About the creators
+          </h2>
+
+          <div className={styles.creator_text_container}>
+            <h3 className={styles.creator_name}>Charlotte</h3>
```

Comment:
Should: The creator_names should be aligned properly with the text underneath

Reply:
Done 👍